activities/ActivityTemplateVue.activity/js/components/presence.js

```diff
@@ -0,0 +1,101 @@
+var Presence = {
+  data: {
+		parent: null,
+    palette: null,
+    presence: null,
+		isHost: false,
+		humane: null
+  },
+  mounted() {
+    this.parent = this.$parent;
+    
+    let vm = this;
+    requirejs(["sugar-web/activity/activity", "sugar-web/env", "sugar-web/graphics/presencepalette", "humane"], function (activity, env, presencepalette, humane) {
+			env.getEnvironment(function (err, environment) {
+				
+				// Handle shared instance
+				if (environment.sharedId) {
+					console.log("Shared instance");
+					vm.presence = activity.getPresenceObject(function (error, network) {
+						if (error) {
+							console.log(error);
+						}
+						network.onDataReceived(vm.onNetworkDataReceived);
+						network.onSharedActivityUserChanged(vm.onNetworkUserChanged);
+					});
+					console.log(vm.presence);
+					vm.parent.presence = vm.presence;
+        }
+        
+			});
+
+			vm.humane = humane;
```

Comment:
I think we could remove humane stuff here. May be we could add a Popup component to handle all humane features?

Reply:
Yes, we can do that.

Reply:
Created the Popup component